finance/sc-im/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem          1.0
+name                sc-im
+version             0.8.2
+license             BSD
+maintainers         nkorth
+
+categories          finance
+description         Spreadsheet Calculator Improvised
+long_description    An ncurses spreadsheet program for terminal
+
+platforms           darwin
+
+homepage            https://github.com/andmarti1424/sc-im
+master_sites        https://github.com/andmarti1424/sc-im/archive
+distfiles           v${version}.tar.gz
```

Comment:
since you're packaging software here for which the source is on GitHub you can make your life much easier by using the [github PG](https://guide.macports.org/chunked/reference.portgroup.html#reference.portgroup.github) since that will do a lot of things for you automatically so you can remove several fields from the Portfile. 

So in this case you could do:
```
github.setup        andmarti1424 sc-im 0.8.2 v
github.tarball_from archives
```

with that you can remove the following fields: `name`, `version`, `homepage`, `master_sites`, `distfiles`, and `worksrcdir`.

Reply:
Thanks, I completely missed the portgroups section of the docs somehow.

I'm still going to keep `worksrcdir` because the makefile is in `src/`, not in the top level.